src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/IProgram.cs

```diff
@@ -9,6 +9,8 @@ public interface IProgram
         Result Result(string query, IPublicAPI api);
         string UniqueIdentifier { get; set; }
         string Name { get; }
+        string Description { get; set; }
         string Location { get; }
+        string SetSubtitle(IPublicAPI api);
```

Comment:
I'll make it a property.

Reply:
The issue with making it a property is that the `Subtitle` depends on `IPublicAPI` and the type of application. The places where the app type is decided do not have access to `IPublicAPI`, this was why I had created a separate function to set the subtitle accordingly and to keep the code neat, instead of passing the API to all functions. 

I made the function private as you suggested and removed it from the interface because functions in the interface must be public while being implemented.

Reply:
Sounds good. Thanks!!